docs/monitoring/autodiscovery.md

```diff
@@ -0,0 +1,60 @@
+title: Sematext Autodiscovery Monitoring
+description: Sematext Autodiscovery Monitoring automatically scans for services that can be monitored using Sematext Agent and, depending on your configuration, automatically sets up monitoring agents. 
+
+## Autodiscovery Monitoring
+
+The centerpiece of Autodiscovery feature is Sematext Agent which scans for services that can be monitoring by Sematext supported integrations. It works in both bare-metal/virtual machine and in container environments. The data about such discovered services is sent to Sematext backend and displayed in UI under Discovery option in the menu on the left.
```

Comment:
be monitoring => be monitored
supported integrations => link to them?

The data about such discovered services is sent to Sematext backend and displayed in UI under Discovery option in the menu on the left. ==> Data about discovered services is displayed under Discovery.